src/main/java/mate/academy/spring/model/User.java

```diff
@@ -14,6 +19,19 @@
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany
```

LAZY is the default value